lib/taskana-core-test/src/test/java/acceptance/task/create/CreateTaskAccTest.java

```diff
@@ -0,0 +1,967 @@
+    assertThat(createdTask.getPrimaryObjRef().getValue())
+        .isEqualTo(defaultObjectReference.getValue());
+    assertThat(createdTask.getExternalId()).isNotNull();
+    assertThat(createdTask.getCreated()).isNotNull();
+    assertThat(createdTask.getModified()).isNotNull();
+    assertThat(createdTask.getBusinessProcessId()).isNotNull();
+    assertThat(createdTask.getClaimed()).isNull();
+    assertThat(createdTask.getCompleted()).isNull();
+    assertThat(createdTask.getModified()).isEqualTo(createdTask.getCreated());
+    assertThat(createdTask.getPlanned()).isEqualTo(expectedPlanned);
+    assertThat(createdTask.getState()).isEqualTo(TaskState.READY);
+    assertThat(createdTask.getParentBusinessProcessId()).isNull();
+    assertThat(createdTask.getPriority()).isEqualTo(defaultClassificationSummary.getPriority());
+    assertThat(createdTask.isRead()).isFalse();
+    assertThat(createdTask.isTransferred()).isFalse();
+
+    Task task2 = createDefaultTask();
+    task2.setExternalId("MyExternalId");
+
+    ThrowingCallable call = () -> taskService.createTask(task2);
+    TaskAlreadyExistException e = catchThrowableOfType(call, TaskAlreadyExistException.class);
+    assertThat(e.getExternalId()).isEqualTo("MyExternalId");
+  }
+
+  @WithAccessId(user = "user-1-2")
+  @Test
+  void should_CreateTask_When_CustomAttributesAreSpecified() throws Exception {
+    Task task = createDefaultTask();
+    Map<String, String> customAttributesForCreate = createSimpleCustomPropertyMap(13);
+    task.setCustomAttributeMap(customAttributesForCreate);
+    Instant expectedReceived = Instant.parse("2019-09-13T08:44:17.588Z");
+    task.setReceived(expectedReceived);
+    Task createdTask = taskService.createTask(task);
+    Instant expectedPlanned = moveForwardToWorkingDay(createdTask.getCreated());
+
+    assertThat(createdTask).isNotNull();
+    assertThat(createdTask.getName()).isEqualTo(defaultClassificationSummary.getName());
+    assertThat(createdTask.getPrimaryObjRef().getValue())
+        .isEqualTo(defaultObjectReference.getValue());
+    assertThat(createdTask.getCreated()).isNotNull();
+    assertThat(createdTask.getModified()).isNotNull();
+    assertThat(createdTask.getBusinessProcessId()).isNotNull();
+    assertThat(createdTask.getClaimed()).isNull();
+    assertThat(createdTask.getCompleted()).isNull();
+    assertThat(createdTask.getModified()).isEqualTo(createdTask.getCreated());
+    assertThat(createdTask.getPlanned()).isEqualTo(expectedPlanned);
+    assertThat(createdTask.getReceived()).isEqualTo(expectedReceived);
+    assertThat(createdTask.getState()).isEqualTo(TaskState.READY);
+    assertThat(createdTask.getParentBusinessProcessId()).isNull();
+    assertThat(createdTask.getPriority()).isEqualTo(defaultClassificationSummary.getPriority());
+    assertThat(createdTask.isRead()).isFalse();
+    assertThat(createdTask.isTransferred()).isFalse();
```

Comment:
This assertions don't seem relevant for the test to me. @gitgoodjhe, what do you think about it? Can we just remove them and only test if createdTask is not null additionally to the customAttributeMap?